common/github.py

```diff
@@ -0,0 +1,98 @@
+# Copyright (c) 2018, Intel Corporation.
```

Comment:
I wonder what copyright should be there and in other newly added files. In `btptester_cron.py` you have
```python
# Copyright (c) 2023, Codecoup.
#
# This program is free software; you can redistribute it and/or modify it
# under the terms and conditions of the GNU General Public License,
# version 2, as published by the Free Software Foundation.
#
# This program is distributed in the hope it will be useful, but WITHOUT
# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
# more details.
#
```
which I think is what it should be, but maybe I'm wrong. @sjanc could you please take a look?

Reply:
Added Codecoup's copyright, I was unsure about what to add as copyright.
Let me know if I should change it again! Thanks